assets/src/apps/delivery/components/TextParser.ts

```diff
@@ -9,21 +9,13 @@ export const templatizeText = (
   isFromTrapStates = false,
 ): string => {
   let innerEnv = env;
-  const vars = extractAllExpressionsFromText(text);
+  let vars = extractAllExpressionsFromText(text);
+  // A expression will not have a ';' inside it. So if there is a ';' inside it, it is CSS and we should filter it.
+  vars = vars.filter((e) => !e.includes(';'));
   /* console.log('templatizeText call: ', { text, vars, state, env }); */
   if (!vars) {
```

Comment:
this line leads me to believe that it's possible for vars to be falsey from above, where we are definitely expecting it to be an array or else the filter call will be an exception